<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCopyRoute</class>
 <widget class="QDialog" name="DialogCopyRoute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy Route</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Company:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QComboBox" name="cbCompany"/>
   </item>
   <item row="6" column="6">
    <widget class="QDateEdit" name="dateEnd">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QDateEdit" name="dateStart">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>EndDate:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Copy route:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Traction Type:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="6">
    <widget class="RouteNameWidget" name="rnw" native="true"/>
   </item>
   <item row="8" column="1" colspan="6">
    <widget class="QLabel" name="lblHelp">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>159</red>
           <green>158</green>
           <blue>158</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>help...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QLineEdit" name="txtRouteName"/>
   </item>
   <item row="7" column="3" colspan="4">
    <widget class="QComboBox" name="cbTractionType"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>StartDate:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>New Company:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="5">
    <widget class="QComboBox" name="cbNewCompany"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RouteNameWidget</class>
   <extends>QWidget</extends>
   <header>routenamewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCopyRoute</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCopyRoute</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
